src/components/Status/Status.js

```diff
@@ -14,17 +15,16 @@ class Status extends Component {
 
   render() {
     const { ocupado } = this.state;
-    const { canChange } = this.props;
+    const { canChange } = !!this.props;
     return (
       <div
-        className="Page"
-        style={{ backgroundColor: ocupado ? 'red' : 'green' }}
+        className={"Page "+(!!ocupado ? ocupado === 'carregando' ? 'carregando' : 'ocupado' : 'livre')}
         onClick={() => {
           if (!canChange) return;
           FirebaseApi.setStatus('banheiros/geral-23', !ocupado);
         }}
       >
-        <p>{ocupado ? 'Tem gente' : 'Tá Livre'}</p>
+        <p>{!!ocupado ? ocupado === 'carregando' ? 'Carregando, por favor aguarde' : 'Tem gente' : 'Está Livre'}</p>
```

Comment:
Again, we don't need the "!!" (normally we would do this if we needed to convert the value of `ocupado` to boolean, which we don't :sweat_smile: ) 
Also, I suggest a set of parenthesis here. Nested ternaries are hard to read =/

```suggestion
        <p>{ocupado ? (ocupado === 'carregando' ? 'Carregando, por favor aguarde' : 'Tem gente') : 'Está Livre'}</p>
```